cmd/parse/gov/proposal.go

```diff
@@ -39,8 +48,16 @@ func proposalCmd(parseConfig *parsecmdtypes.Config) *cobra.Command {
 			// Get the database
 			db := database.Cast(parseCtx.Database)
 
+			// Build expected modules of gov modules for handleParamChangeProposal
+			distrModule := distribution.NewModule(sources.DistrSource, parseCtx.EncodingConfig.Marshaler, db)
+			mintModule := mint.NewModule(sources.MintSource, parseCtx.EncodingConfig.Marshaler, db)
+			slashingModule := slashing.NewModule(sources.SlashingSource, parseCtx.EncodingConfig.Marshaler, db)
+			stakingModule := staking.NewModule(sources.StakingSource, slashingModule, parseCtx.EncodingConfig.Marshaler, db)
+
 			// Build the gov module
-			govModule := gov.NewModule(sources.GovSource, nil, nil, nil, nil, nil, parseCtx.EncodingConfig.Marshaler, db)
+			govModule := gov.NewModule(
+				sources.GovSource, nil, distrModule, mintModule, slashingModule, stakingModule, parseCtx.EncodingConfig.Marshaler, db,
+			)
```

Comment:
Instead of building the modules by hand, can't we simple call the `BuildModules` function?

Reply:
thanks @RiccardoM , would you mind reviewing again? thanks

Reply:
Ok the new code looks ugly. It was a lot better the one before honestly